pages/super-collaborator/run.html

```diff
@@ -172,6 +173,18 @@
         hpccWasm.graphviz.layout(window.dot, "svg", "dot").then(svg => {
           target.innerHTML = svg;
           drawing = false
+          const targetBounds = { width: document.querySelector('#target').clientWidth, height: document.querySelector('#target').clientHeight } 
+          const svgBounds = { width: document.querySelector('#target svg').clientWidth, height: document.querySelector('#target svg').clientHeight }
+          // console.log(targetBounds, svgBounds)
+          let svgElement = document.querySelector('#target svg')
+          let panSVG = svgPanZoom(svgElement)
+          document.querySelector('#target svg').style.height = "100%"
+          document.querySelector('#target svg').style.width = "100%"
+          if (targetBounds.width < svgBounds.width || targetBounds.height < svgBounds.height) {
+            panSVG.resize()
```

Comment:
If the SVG is larger than the target, resize it so that it will fit within the target